Makefile

```diff
@@ -37,6 +41,9 @@ test: gofmt terraform-fmt vet lint
 	go mod tidy
 	go test $(TEST_PKGS) $(TEST_FLAGS)
 
+test-tags:
```

Comment:
For my understanding how exactly this will work?

Reply:
For example, if I want to run acceptance test for cluster group alone. I need to set the required env variables and BUILD_TAG=clustergroup.

Then run "make test-tags"

Reply:
Ok now I see BUILD_TAG is updated have all build tags. That's fine, initially it was empty and that's where the confusion raised. Thanks for taking care of it